docs/alternator/alternator.md

```diff
@@ -1,5 +1,8 @@
 # Alternator: DynamoDB API in Scylla
 
+
+**Alternator doesn't support Tablets, because it relies on LWT.**
```

Comment:
I tried to make this warning look better, but I couldn't. What I tried was:
- referencing other pages (like I do in other docs changes)
- making a note out of it by "> [!NOTE]"
- adding emojis, e.g. ":warning:"
- other ideas
but none worked, so I left it as is.

Reply:
@annastuchlik does this "..note.." thing work in .md (this file is not an .rst...)?

Anyway, I'm not this warning is needed - it's not that Alternator does not "support" tablets - it just doesn't use them. It's not like CQL where you create the keyspace yourself and accidentally do it with tablets and then discover LWT or counters aren't supported - in Alternator, it creates the keyspace for you and does it without tablets.

I'm worried that people will see this message as the **first** line of the Alternator document, and have no idea what it means.

Reply:
I agree with @nyh that this piece of info may not be that important for Alternator users, hence its "importance" has been degraded and I did something similar to what's been suggested - removed the warning note from the doc's beginning and just added a sentence to the `Alternator design and implementation` section which says:
> Currently, Alternator doesn't use Tablets. That's because Alternator relies
on LWT (lightweight transactions), and LWT is not supported in keyspaces
with Tablets enabled.